### ISSUE TITLE
Critical credentials leak

### ISSUE THREAD

**Issue by BilousVladyslav** [nazi-k/steamspeculate#1]
Hello, @nazi-k 
I found **all** possible credentials for your Steam account in your [last commit](https://github.com/nazi-k/steamspeculate/commit/4904f8171aa0c81d6d5cb6ab4a732d791b995a6b). I was really surprised by the fact that it was so easy to get, so I checked if they work and unfortunately they do:
- The API key is still valid, so it's possible to create a trade offer without your permission.
- The revocation code is present in the `steamguard.json` file, which allows you to re-create the Steam Guard without your permission.

I understand that these credentials are required to use the `steampy` library, but you don't need to use the entire `maFile` from the SteamDesktopAuthenticator (check the [steampy documentation](https://github.com/bukson/steampy#usage)), and you should never store any credentials inside a repository (regardless of whether it's private or public).

I highly recommend that you create a new API key for your account and remove the Steam profile credential commit. Changing your password will **not** help secure your account.

[Here you can change your Steam API key](https://steamcommunity.com/dev/apikey)
[How to delete a latest commits from git history](https://stackoverflow.com/a/23205409)

**Comment by woctezuma** [comment]
Good catch. For info, [the account seems dummy](https://steamcommunity.com/profiles/76561199079793343), it is a level-1 account, which means it is not a limited account, but the only game is Dota2.
